Fibonacci.java

```diff
@@ -0,0 +1,18 @@
+public class Fibonacci {
+
+	public static void main(String[] args) { 
+		int nthFib = Integer.parseInt(args[0]);
+		System.out.println(new Fibonacci().findNthFib(nthFib));
```

Comment:
Does this Fibonacci have a toString() function? Maybe store the result in a varible and then print it after